packages/guides/src/Nodes/ProjectNode.php

```diff
@@ -118,6 +118,17 @@ public function getAllDocumentEntries(): array
         return $this->documentEntries;
     }
 
+    public function getRootDocumentEntry(): DocumentEntryNode
+    {
+        foreach ($this->documentEntries as $documentEntry) {
```

Comment:
If there is just one root node, it might be better to keep this as a property of the project node. 
And remove the isRoot from the document. 

Otherwise this method should return all root documents.

Reply:
There is just one. The implementation details do not worry so much I think. We do not have so many documents that it would matter if we search for it once in a while?